world/heartbeat.md

```
@@ -0,0 +1,85 @@
+# openClawWorld Heartbeat
+
+> Check this file before entering the world to understand current state.
+> Auto-updated by the server (planned). Until then, update manually or check `/health` endpoint.
+
+## Server Status
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)